### ISSUE TITLE
Authentication in 3.0 is behind the Authentication in 2.4

### ISSUE THREAD

**Issue by johann8384** [OpenTSDB/opentsdb#1018]

**Comment by IDerr** [comment]
What do you mean by behind ?

**Comment by johann8384** [comment]
It appears (at fist glance) that the Auth code in 3.0 is closer to my original PR for it, but 2.4.0 has some additional enhancements, perhaps done by Chris. I noticed when I updated my Authentication plugin for 2.4.0 and then saw it was no longer compatible with 3.0.

For example, in 2.4.0 the code uses an AuthStatus class but in 3.0 that is not present.

**Comment by IDerr** [comment]
Oh in that way. Ok thanks for the explanation !

**Comment by IDerr** [comment]
I'll try to implement kerberos auth before the 2.4 milestone, but I'm not sure that I will time.

If I can't, I will wait your PR for 3.0.

**Comment by johann8384** [comment]
I don't think we need to do that, 2.4 is fine as-is, I am working on  supporting authorization as well as authentication in 3.0, as the 2.x series plugins only do authentication
